blog/poc3.md

```diff
@@ -0,0 +1,127 @@
+The POC3 code is now merged into the official [go-ethereum repository's master branch](https://github.com/ethereum/go-ethereum).
+It has been a year and a half since the first release of the POC2 series was deployed and [the swarm project launched its public alpha network](https://blog.ethereum.org/2016/12/15/swarm-alpha-public-pilot-basics-swarm/).
+Two [swarm summits](https://swarm-gateways.net/bzz:/swarm-summit.eth), two [orange papers](http://swarm-guide.readthedocs.io/en/resources.html#orange-papers) and [forty thousand lines of code](https://github.com/ethereum/go-ethereum/pull/17041) later, it is time to give an update.
+
+## Basics of Swarm
+
+Swarm content storage is easiest described to not-so-technical audience as "bittorrent on steroids". Here we venture into a more elaborate technical presentation.
+In an earlier blog post, we introduced the basics of swarm storage and content distribution. To set the scene for a broader scope, here we recap the most important points.
+
+Swarm is a service offers APIs to upload and download content and through a url-based addressing offers virtual hosting of web-sites and dapps without webservers, using a peer-to-peer distributed infrastructure.
+When content is uploaded to Swarm it is chopped up into pieces called chunks.
+Each chunk is identified with a unique address deterministically derived from its content (using the chunk hash).
+The references of data chunks are themselves packaged into a chunk which in turn has its own hash. In this way the content gets mapped into a merkle tree the root hash of which, called the Swarm hash is used as a reference to a file. When this reference then is used to retrieve the file, the client can recursively unpack the tree by fetching the individual chunks and assemble the original content. This hierarchical merklised swarm hash construct offers integrity protected random access into large files; the thing you do you skip to parts of video or look up a key in a database file.
+
+Most importantly however, this means that on the network layer, swarm only needs to take care of storing and retrieving chunks.
+Swarm is a peer-to-peer network of nodes running the swarm client. The network uses ethereum's devp2p/rlpx networking layer to run the bzz suite of protocols. Nodes are identified with a swarm address deterministically derived from the node's ethereum account public key, which can place them in the same address space as the chunks themselves, so we can define proximity between nodes and chunks.
+The semi-permanent TCP connections between peers define an overlay network where messages can be relayed from node to node.
+The Swarm network implements a :dfn:`distributed preimage archive` (DPA), which is essentially a specific type of content addressed distributed chunk store;
```

Comment:
are you going to deploy this on RTD? the links above are in MD format but this is RST